x-pack/test/security_api_integration/tests/session_concurrent_limit/global_limit.ts

```diff
@@ -217,6 +223,9 @@ export default function ({ getService }: FtrProviderContext) {
           .set('kbn-xsrf', 'xxx')
           .set('Cookie', basicSessionCookie.cookieString());
         statusCodes.push(statusCode);
+        await setTimeoutAsync(1000);
+        await es.indices.refresh({ index: '.kibana_security_session*' });
+        await setTimeoutAsync(2000);
```

Comment:
@azasypkin This was the only spot where just the index refresh was not enough. I had to put the static delays back in. Curious if you have any ideas on why the refresh wouldn't work here as it does elsewhere.

Reply:
Hmmm, I have no idea to be honest, but let me think a bit.

Reply:
By the way shouldn't you call this refresh before this `for` loop, right after you created sessions?

Reply:
Right...that was a bit silly of me. I had moved the delays around previously and sort of lost the plot on this one. In the right spot it works 😅 . Thanks!